src/Cms/Auth.php

```diff
@@ -79,13 +80,11 @@ public function currentUserFromBasicAuth(BasicAuth $auth = null)
             throw new PermissionException('Basic authentication is only allowed over HTTPS');
         }
 
-        if ($user = $this->kirby->users()->find($auth->username())) {
-            if ($user->validatePassword($auth->password()) === true) {
-                return $user;
-            }
+        try {
+            return $this->validatePassword($auth->username(), $auth->password());
+        } catch (Throwable $e) {
```

Comment:
Same as above. If we want to have a way to get proper error messages on failed logins, we need to let the exceptions bubble up.

Reply:
That one was like this before as well. But I will see if I can improve on this further.

Reply:
Fixed now (not yet pushed).